src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -40,7 +40,7 @@ case class WorkspaceRequest (
 case class Workspace (
                       namespace: String,
                       name: String,
-                      bucketName: String,
+                      workspaceId: String,
```

Comment:
We can tear up the last few commits and go back to bucketName in the Workspace if you'd prefer.